os/dbuild.sh

```diff
@@ -46,6 +46,24 @@ if ! which docker > /dev/null; then
 	exit 1
 fi
 
+KERNEL_DBG_BIN=
+APP1_DBG_BIN=
+APP2_DBG_BIN=
+CMN_DBG_BIN=
+function FIND_DBGBINFILE()
+{
+	APP1_DBG_BIN="app1_dbg"
```

Comment:
Is it ok to hardcode the binary names or should we fetch them from the partition list ?